cypress/integration/article_spec.js

```diff
@@ -0,0 +1,24 @@
+const url = 'http://localhost:7080/';
```

Comment:
Remove the trailing `/` from this URL and add it wherever you are adding the path. This is to give greater control over the base URL and path.

Reply:
Will do. In the following PR this will most likely change anyway